topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -170,140 +157,297 @@ To estimate sequence quality along all sequences, we now use [FastQC](https://ww
+> > ### {% icon solution %} Solution
+> > 1. The mean score over the sequence is dropping at the end of the sequences. It is usual: the sequencers are incorporating more errored nucleotides at the end. But the score stays good: over 28.
+> > 2. The box plots are getting wider at the end of the sequences. It means a lot of sequences have their score dropping at the end of the sequence. At after 31 nucleotides, more than 10% of the sequences have scores below 20.
+> >
+> {: .solution }
+{: .question}
+
+When the median quality is to a phred score of ~20, we should consider trimming the sequence at that point, since sequence with lower quality than that is likely to cause more problems than it fixes. We will explain that process in the next section
+
+FastQC produces other diagnostic plots to assess sample quality.
+
+## Per sequence quality scores
+
+It plots the average quality score over full length of reads on the x-axis and the total number of reads with this score on the y-axis:
+
+![Per sequence quality scores](../../images/quality-control/per_sequence_quality_scores.png "Per sequence quality scores")
+
+The distribution of average read quality should be fairly tight in the upper range of the plot. It can also report if a subset of the sequences have universally low quality values: it can happen because some sequences are poorly imaged (on the edge of the field of view etc), however these should represent only a small percentage of the total sequences.
+
+## Per base sequence content
+
+![Per base sequence content](../../images/quality-control/per_base_sequence_content.png "Per base sequence content for a DNA library")
+
+Per Base Sequence Content plots out the percentage of each of the four nucleotides (T, C, A, G) at each position across all reads in the input.  As for the per base sequence quality, the X-axis is non-uniform.
+
+In a random library we would expect that there would be little to no difference between the four bases. The proportion of each of the four bases should remain relatively constant over the length of the read with `%A=%T` and `%G=%C`, and the lines in this plot should run parallel with each other. 
+
+> ### {% icon details %} Biases by library type
+> 
+> It's worth noting that some types of library will always produce biased sequence composition, normally at the start of the read. Libraries produced by priming using random hexamers (including nearly all RNA-Seq libraries as in the previous plot), and those which were fragmented using transposases, will contain an intrinsic bias in the positions at which reads start (the first 10-12 bases). This bias does not involve a specific sequence, but instead provides enrichment of a number of different K-mers at the 5' end of the reads. Whilst this is a true technical bias, it isn't something which can be corrected by trimming and in most cases doesn't seem to adversely affect the downstream analysis. It will, however, produce a warning or error in this module.
```

Comment:
types of library -> library types

Reply:
Also a funny effect that I found recently is that the phred score is very low if the first nucleotides are always the same (e.g. in RADSeq where a restriction enzyme always cuts at a certain pattern). Then the complete flow cell always sees the same nucleotide which confuses the error model of the base caller.

Reply:
Should we mention that here? How would you phrase it?

Reply:
Maybe something along the following lines: 

"""
An extreme example is the RADSeq protocol where fragmentation is done with restriction enzymes that cut at a specific short sequence motif, i.e. each fragment starts with the same nucleotides which are processed in lock step by the sequencer. Thus, the sequencer gets the same signal for the complete lane or flow cell while processing these nucleotides. Since the error model of the base caller can't cope with this case it assigns very low quality values to these sequences.
"""

ping @yvanlebras do you agree with this?